src/main/java/rs/teslaris/core/controller/PrizeController.java

```diff
@@ -84,4 +85,13 @@ public void deletePrizeProof(@PathVariable Integer prizeId,
                                  @PathVariable Integer proofId) {
         prizeService.deleteProof(proofId, prizeId);
     }
+
+    @PatchMapping("/merge/person/source/{sourcePersonId}/target/{targetPersonId}")
+    @PreAuthorize("hasAuthority('MERGE_PERSON_METADATA')")
+    public void switchInvolvementsToOtherPerson(@PathVariable Integer sourcePersonId,
```

Comment:
Again, why this is not part of MergeController? Moreover, the name of the method should be corrected to switchPrizesToOtherPerson

Reply:
Save answer as above :smile:

Reply:
ok, probably we will move this (and don't forget to rename this one)

Reply:
Renamed and refactored :smile: